functions.php

```diff
@@ -0,0 +1,20 @@
+<?
+/* displays list of DB info
+ *
+ * @param $winners array
+ *
+ * returns string of all winners
+ */
+function display_winners(array $winners) :string
+{
+    $output = '';
+foreach ($winners as $winner) {
+    $output .= '<h2>Prize year: ' . $winner['prize_year'] . '</h2>';
+    $output .= '<span>Author: ' . $winner['author_name'] . '</span><br />';
+    $output .= '<span>Title: ' . $winner['book_name'] . '</span><br />';
+    $output .= '<span>Author nationality: ' . $winner['author_nationality'] . '</span><br />';
```

Comment:
I'm undecided on these br tags.

Technically they are designed for spacing out text, so this isn't terrible. I just feel weird about seeing them! If you used block level elements like divs then they would naturally break onto separate lines